src/main/java/seedu/address/logic/commands/ListLogsCommand.java

```diff
@@ -36,20 +40,38 @@ public ListLogsCommand(IdentityNumber id) {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        //TODO: Handle behaviour later
         requireNonNull(model);
-        model.updateFilteredLogListById(this.identityNumber);
 
+        ObservableList<Person> personList = model.getAddressBook().getPersonList();
+        int personIndex = -1;
+        Person person = null;
+
+        for (Person p: personList) {
```

Comment:
Suggestion: personIndex = personList.indexOf(p) can be replaced with personIndex = i; if you iterate using a for loop with an index.